### PR TITLE
CHE-10360: Hide the 'From Archetype' field  if parent folder with pom.xml is selected

### PR DESCRIPTION
**Pull request by vinokurig**
### What does this PR do?
Hide the 'From Archetype' input field in the Maven project type wizard if 'Parent' witch contains `pom.xml` is selected in the import project Wizard due to the maven archetype generation plugin restrictions.

### What issues does this PR fix or reference?
#10360

#### Release Notes

Fix bug with the maven archetype and selected (non root) project path.

#### Docs PR

N/A

### CODE REVIEW

**Comment by vinokurig** [comment]
ci-test

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/569/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/569//Selenium_20tests_20report/)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/10718)
docker image: eclipseche/che-server:10718
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by vinokurig** [comment]
ci-test

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/574/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/574//Selenium_20tests_20report/)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/10718)
docker image: eclipseche/che-server:10718
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

File: plugins/plugin-maven/che-plugin-maven-ide/src/main/java/org/eclipse/che/plugin/maven/client/wizard/MavenPagePresenter.java [hunk 1]
```diff
@@ -212,11 +216,25 @@ public void go(AcceptsOneWidget container) {
       }
       updateDelegate.updateControls();
     }
+    projectService
+        .getItem(Path.valueOf(dataObject.getPath()).parent().append("pom.xml"))
+        .then(
+            result -> {
+              updateView(container, false);
```
Comment (vparfonov) [inline plugins/plugin-maven/che-plugin-maven-ide/src/main/java/org/eclipse/che/plugin/maven/client/wizard/MavenPagePresenter.java]: I think need to add some notification here, user must have information why create action disabled

**Review by vparfonov — APPROVED** [review]

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/614/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/614//Selenium_20tests_20report/)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/10718)
docker image: eclipseche/che-server:10718
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by vinokurig** [comment]
ci-test

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/635/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/635//Selenium_20tests_20report/)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/10718)
docker image: eclipseche/che-server:10718
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Review by musienko-maxim — APPROVED** [review]

**Comment by vinokurig** [comment]
ci-build

**Pull request merged**